toolchain/check/testdata/facet/no_prelude/access.carbon

```diff
@@ -8,6 +8,14 @@
 // TIP: To dump output, run:
 // TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/check/testdata/facet/no_prelude/access.carbon
 
+// --- core.carbon
+package Core;
+// TODO: switch to min_prelude
```

Comment:
Do you intend to do this in this PR? If so, here's a reminder to do so :)

Reply:
No, that seemed disruptive. I plan to do a bunch all at once in a dedicated PR.